packages/agent/src/tool/user-input/UserInputTool.ts

```diff
@@ -0,0 +1,63 @@
+import zod from "zod";
+import { createToolAction } from "../ToolAction";
+import { ExecuteToolFunction } from "../ExecuteToolFunction";
+import { FormatResultFunction } from "../../action";
+import readline from "readline";
+
+export type UserInputInput = {
+  query: string;
+};
+
+export type UserInputOutput = {
+  text: string;
+};
+
+export const getUserInput = ({
+  id = "user-input",
```

Comment:
The id will also be shown to the LLM. I found that IDs that are more "readable" can be better for the LLM (because of the underlying next word prediction). E.g. `ask-user` might lead to the LLM picking the action more often.